<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Alert</class>
 <widget class="QWidget" name="Alert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>444</width>
    <height>106</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>87</green>
        <blue>83</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>87</green>
        <blue>83</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>87</green>
        <blue>83</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>85</red>
        <green>87</green>
        <blue>83</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Alert</string>
  </property>
  <property name="windowIcon">
   <iconset theme="clock">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>401</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">color: rgb(238, 238, 236);</string>
      </property>
      <property name="text">
       <string>Delay(min):</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cbDelay">
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="maxCount">
       <number>10</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btDelay">
      <property name="text">
       <string>Delay</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btRestart">
      <property name="text">
       <string>Restart</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>20</y>
     <width>401</width>
     <height>26</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(239, 41, 41);</string>
      </property>
      <property name="text">
       <string>WARNING(TIME OUT)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbTimer">
      <property name="font">
       <font>
        <pointsize>16</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(138, 226, 52);</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btRestart</sender>
   <signal>clicked()</signal>
   <receiver>Alert</receiver>
   <slot>restart()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>364</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>364</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btDelay</sender>
   <signal>clicked()</signal>
   <receiver>Alert</receiver>
   <slot>delay()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>delay()</slot>
 </slots>
</ui>
